run-tests.js

```diff
@@ -524,11 +524,16 @@ ${ENDGROUP}`)
       if ((!passed || shouldContinueTestsOnError) && isTestJob) {
         try {
           const testsOutput = await fs.readFile(`${test}${RESULTS_EXT}`, 'utf8')
+          const obj = JSON.parse(testsOutput)
+          obj.processEnv = {
+            NEXT_TEST_MODE: process.env.NEXT_TEST_MODE,
+            HEADLESS: process.env.HEADLESS,
+          }
```

Comment:
@ijjk Is there a better way to capture these besides parse and stringify?

Reply:
If we're wanting to inject into this payload then this works

Reply:
Ok cool. I wasn't sure if there was a way to emit env var data from the cli command that was creating the file.